docs/flatlist.md

```diff
@@ -116,47 +116,42 @@ const DATA = [
   },
 ];
 
-const Item = ({ id, title, selected, onSelect }) => {
-  return (
-    <TouchableOpacity
-      onPress={() => onSelect(id)}
+const App = () => {
+  const [selected, setSelected] = useState({ id: null });
+
+  const onSelect = (item) => {
+    setSelected(item);
+  }
+
+  const renderItemSelect = (item) => {
+    return item.id === selected.id ? (
+       <TouchableOpacity
+      onPress={() => onSelect(item)}
       style={[
         styles.item,
-        { backgroundColor: selected ? '#6e3b6e' : '#f9c2ff' },
```

Comment:
Please use ternary operator for the `backgroundColor` like in the earlier version of example.